tendrl/monitoring_integration/flows/delete_resource_from_graphite/__init__.py

```diff
@@ -6,7 +6,9 @@
 from tendrl.commons import flows
 from tendrl.commons.utils import etcd_utils
 from tendrl.commons.utils import log_utils as logger
-from tendrl.monitoring_integration.grafana import create_dashboards
+from tendrl.monitoring_integration.grafana import utils
+
+CONF_PATH = "/etc/tendrl/monitoring-integration/carbon.conf"
```

Comment:
This file is used by carbon-cache only, we are keeping this conf file in monitoring_integration to replace the carbon-cache configuration file.
when monitoring_integration install it will replace the original carbon.conf file by this

So i am thinking like if user change path directly inside /etc/carbon/carbon.conf then deletion of graphite data won't happen so better i read graphite data path from /etc/carbon/carbon.conf